### ISSUE TITLE
tab: navigation issues with screen readers

### ISSUE THREAD

**Issue by EamonKavanagh** [angular/material#10895]
**Actual Behavior**:
 -  mdTab components set the tab-index of all tabs except the first to -1.

When focusing on an md-tab component, the screenreader I tested (ChromeVox) announces that you are now in a tab list, which tab you are on, and the total number of tabs. To navigate through the tab list, the user must use the left/right arrows. This puts the burden of knowledge on the user, in terms of knowing to do this (much like with a list box and up/down arrows). Is there some form of documentation that indicates that this expectation is built into the design and accessible?

**CodePen** (or steps to reproduce the issue): *
- See the Angular mdTab demo here: https://material.angularjs.org/latest/demo/tabs

**Comment by Splaktar** [comment]
On VoiceOver for macOS, when you land on the first tab, after the basic tab name, tab position, and number of tabs, it explains to you "You are currently on a tab, 1 of 3. To select this option, press Control-Option-Space." However, it does not tell the user how to navigate the tabs. On Chromevox, it doesn't even tell the user how to activate the selected tab.

**Comment by Splaktar** [comment]
Looking into this a bit, I found that we do most things covered in [this article](http://simplyaccessible.com/article/danger-aria-tabs/), but we don't set `aria-hidden` on the `md-tab-content` elements which are the `tabpanel`s.

This article goes on later to describe the exact issue faced here. How are users supposed to know to use the arrow keys for navigation of the tabs?

However, the article goes on to suggest removing the `role="tablist"`, `role="tab"`, `role="tabpanel"`, and `tabindex="-1"` attributes while handling focus management manually. This seems like a pretty big breaking change for us at this point, so I'm going to explore alternative solutions

**Comment by Splaktar** [comment]
Our implementation best matches the [Tabs with Manual Activation Example](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html) which does document the left and right arrow behavior. However, this still does not mean that it is obvious to users.

**Comment by Splaktar** [comment]
Another missing piece that we don't have, but is found in the [WAI-ARIA Authoring Practices for Tab Panel](https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel):
- Each element with role `tab` has the property `aria-controls` referring to its associated `tabpanel` element.

This document also covers the expected interaction via the right/left arrow keys.

**Comment by Splaktar** [comment]
Submitted PR https://github.com/angular/material/pull/11402 to improve this situation.

**Comment by Splaktar** [comment]
@bryan-sullivan-ascendlearning-com thank you for your feedback. I wasn't attempting to say that we match it perfectly. If you've got a bug to report, please open a new issue.
